vendor/milesight-iot/em500-smtc.yaml

```diff
@@ -0,0 +1,85 @@
+name: EM500-SMTC
+description: Soil Moisture, Temperature and Electrical Conductivity Sensor
+
+firmwareVersions:
+  - version: '2.30'
+    numeric: 1
+    hardwareVersions:
+      - 'V1.x'
+
+    profiles:
+      EU863-870:
+        id: profile-eu868
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      US902-928:
+        id: profile-us915
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      AU915-928:
+        id: profile-au915
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      AS923:
+        id: profile-as923
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      CN470-510:
+        id: profile-cn470
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      KR920-923:
+        id: profile-kr920
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      IN865-867:
+        id: profile-in865
+        lorawanCertified: true
+        codec: em500-smtc-codec
+      RU864-870:
+        id: profile-ru864
+        lorawanCertified: true
+        codec: em500-smtc-codec
+
+sensors:
+  - battery
+  - temperature
+  - moisture
+  - EC

```

Comment:
please comment EC sensor

Reply:
EC should be conductivity. 
[EM500-SMTC.zip](https://github.com/TheThingsNetwork/lorawan-devices/files/6573131/EM500-SMTC.zip)